modules/docker-compose.perception.yaml

```diff
@@ -30,7 +31,8 @@ services:
               capabilities: [ gpu ]
     command: /bin/bash -c "ros2 launch camera_object_detection eve_launch.py"
```

Comment:
on startup, this means that we only launch the generic object detectors for Eve. Do we also want to launch the traffic light detector? If so, I would combine the launch files of eve_launch and traffic_signs_launch

Reply:
@Gongsta @Miniapple8888 thoughts on this before we merge?

Reply:
i think we should combine them (and do the same for traffic light detection) since both are using camera object detection anyways. I'll make that change in the code.